CHANGES.txt

```diff
@@ -13,6 +13,8 @@ Unreleased
 
 - Adjusted connect arguments to accept credentials within the HTTP URI.
 
+- Adjusted connect arguments to enable SSL using SQLAlchemy DB URI.
```

Comment:
Maybe rephrase this to make it more end-user comprehensible

Reply:
Good idea. I tried to rephrase it with 9984cc4826.